sal_interfaces/GenericCamera/GenericCamera_Events.xml

```diff
@@ -794,4 +794,12 @@
     <Subsystem>GenericCamera</Subsystem>
     <EFDB_Topic>GenericCamera_logevent_ccsConfigured</EFDB_Topic>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>GenericCamera</Subsystem>
+    <EFDB_Topic>GenericCamera_logevent_startAutoExposure</EFDB_Topic>
```

Comment:
How about: `autoExposureStarted`? Also, might be a good idea to have the parameters passed to the command sent here, so the user can check that what it send is what it got.

Reply:
I modified it to `autoExposureStarted` but note that I was following the way the live view events have been implemented and there it is `startLiveView` so we may want to modify that one as well. I added the parameters that were passed on with the corresponding command.